andela/AndelaEatsUI
# #161728073 Refactor vendor endpoints

**What does the PR do?**
- This PR `Refactor's vendor endpoints`

**Description of Task to be completed?**
- refactor vendor endpoints
- change variable input names
- refactor vendor action
- update vendor action test file

**How should this be manually tested?**
- Clone the repository [here](https://github.com/andela/AndelaEatsUI.git)
-  cd into AndelaEatsUI and checkout to this branch `fix-refactor-vendor-endpoint-161728073`
- run these commands `npm run bundle` and `npm run dev` 
- Log in as an `Admin`, navigate to `admin dashboard`, select `Vendor`
- run the command `npm run test-dev` to ensure all tests are passing

**What are the relevant pivotal tracker stories?**
- [#161728073](https://www.pivotaltracker.com/n/projects/2178749/stories/161728073)

**Screenshots**
![screen shot 2018-11-05 at 8 21 47 pm](https://user-images.githubusercontent.com/21138774/48021299-80cae980-e138-11e8-95ea-dabb1d92507e.png)
